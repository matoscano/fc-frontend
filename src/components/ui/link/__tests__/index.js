import React from 'react';
import { render } from '@testing-library/react';
import Link from '../index';

describe('Link', () => {
  it('renders correctly', () => {
    const { asFragment } = render(<Link href="/" />);
    expect(asFragment()).toMatchSnapshot();
  });
});
