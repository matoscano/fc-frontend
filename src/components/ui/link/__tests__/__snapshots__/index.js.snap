// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Link renders correctly 1`] = `
<DocumentFragment>
  <a
    class="sc-AxjAm dpQVlV"
    href="/"
  />
</DocumentFragment>
`;
